ProjectSettings/MemorySettings.asset

```diff
@@ -0,0 +1,35 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!387306366 &1
+MemorySettings:
```

Comment:
Do we need this file?

Reply:
We might not need it but it is generated by Unity. It is not included into the `.unitypackage`.